TestTables/CMIP6_coordinate.json

```diff
@@ -1290,7 +1268,7 @@
             "type": "double", 
             "valid_max": "", 
             "valid_min": "", 
-            "value": "84000.", 
+            "value": "m", 
```

Comment:
@taylor13 is that right?

Reply:
@taylor13 interestigly this does not show up on cmip6-tables changes because it appears to have been introduce in26. The actual cmor tables in the code are still at 24, which is why we notice it here but in the cmip6-cmor-tables.

Reply:
This is definitely an error in the table.  value *should* be 84000.  Can you edit the CMOR dimension table by hand before committing?  I'll inform Martin it needs to be corrected for release 28.